### PR TITLE
Switch to msbuild

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/aspnet/Coherence/issues/167

### CODE REVIEW

File: src/CoherenceBuild/CoherenceBuild.csproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.0.1" />
+    <PackageReference Include="NuGet.Packaging" Version="3.5.0" />
```
Comment (natemcmaster) [inline src/CoherenceBuild/CoherenceBuild.csproj:10]: Update to 4.0.0? Latest on NuGet.org is 4.0.0-rtm-2265.

**Review by natemcmaster — APPROVED** [review]
Looks good. One minor suggestion.

File: src/CoherenceBuild/CoherenceBuild.csproj (hunk 1 above)
Comment (pranavkm) [reply src/CoherenceBuild/CoherenceBuild.csproj:10]: I'm working on removing the dependency in a follow up - https://github.com/aspnet/Coherence/issues/170.

**Comment by natemcmaster** [comment]
Ok. :shipit:

**Pull request merged**